lib/permutations.rb

```diff
@@ -1,4 +1,23 @@
 
 def permutations?(string1, string2)
```

Comment:
This will fail for `heelo` and `hello`.  You should also count the number of each letter in each string.